webrender/src/resource_cache.rs

```diff
@@ -127,10 +136,11 @@ where
         }
     }
 
-    fn get(&self, key: &K) -> &V {
-        self.resources
-            .get(key)
-            .expect("Didn't find a cached resource with that ID!")
+    /// In some cases, we can't handle the resource request(e.g. the user
```

Comment:
I think we should still differentiate between keys that didn't end up in the map for some other logical reasons, and keys that were just failing to be added. Perhaps, the resources map should contain `Result<V>` instead of `V`, so that we can handle those failures in a more explicit way?